### PR TITLE
Add Contributor Name

### PR DESCRIPTION
**Pull request by LieselWong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,4 @@ Registration System for BigRed//Hacks
 ## Contributors
 **Current Contributors**
 - **Colin Wu** - Software Lead
+- **Liesel Wong** - Software Member
```
Comment (CollinWoo) [inline README.md]: I think Software Developer would be a more descriptive role name

**Comment by LieselWong** [comment]
waaasup

**Comment by CollinWoo** [comment]
asdfakljsdjlkadsl

**Review by CollinWoo — CHANGES_REQUESTED** [review]

**Review by CollinWoo — APPROVED** [review]
Welcome to the team! 🎉

**Pull request merged**